### DISCUSSION TITLE
Trying revoke the SAML Identity link for a user

### DISCUSSION THREAD

**Discussion by Awnhumbla** [community/community#116804] [New to GitHub]
I'm org owner and I'm trying to revoke a GitHub user account's link to the SAML IdP. The user needs to have a new GitHub user account linked since they've lost access to the old user account - but when they try to link with the new one, they receive this message:

> There was an issue joining the organization: Your GitHub user account @UserName is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'UserEmail' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance

Usually, I can revoke a user account's SAML link by going to the user, clicking here: 
![image](https://github.com/community/community/assets/91959338/62c52547-0bdb-4a2e-a709-5a597eae83ca)
and then click **revoke**

But this user account I'm trying to revoke from is only _invited_ to the organization, not a member yet since they cannot access the old user account anymore - hence they cannot accept the invitation. 

...and I cannot revoke the SAML IdP since I cannot see it in the organization. 🤷‍♂️ 

Feels like I'm stuck in limbo here - anybody have a clue what I can do??

**Comment by Awnhumbla** [comment]
It's been two weeks with no reply - I hope someone sees this at some point 😄

**Comment by rajivranjannice** [comment]
Export all user information from the organization and check the affected user's email address is linked with another user profile. I am pretty sure it will be linked with some other profile.
Go to https://github.com/orgs/<your_org>/people/<old_user_account>/sso SSO Identity Link, and unlink the account.

**Reply by Awnhumbla** [reply]
Thanks for the reply - we have fixed  the problem now since we have moved the whole thing to a new enterprise and changed the SAML setup - things seem to work now.

**Reply by julienbonastre** [reply]
Wow, drastic move to fix a user 🤣

**Comment by Awnhumbla** [comment]
I have updated the SAML setup and things work now.

**Comment by swapnilguptaTW** [comment]
Similar issue as above and on the above mentioned link "https://github.com/orgs/<your_org>/people/<old_user_account>/sso" I do not see any SSO acount linked.
<img width="881" alt="Screenshot 2024-09-04 at 1 16 28 PM" src="https://github.com/user-attachments/assets/efb66f07-13c7-4ead-9151-83be410f6b7c">

**Comment by brunojppb** [comment]
It took me a while to figure this out, but I had a similar issue with a new joiner at our org and his company email was linked to some other account I could not see among our users.

Here is what I did to find out:

- [x] Head to `https://github.com/orgs/<your_org>/people`
- [x] On the top-right corner, you will see an `export` button to export all users from your org. Click there and select `JSON`
- [x] Now open the JSON file on the editor of your choice (e.g. VSCode)
- [x] Now search for the email that is wrongly associated with another account

You should see something like the following:

![CleanShot 2024-09-04 at 13 20 46@2x](https://github.com/user-attachments/assets/3d609b99-b21e-4e25-a8f1-210003c631cc)

on the `"login"` property, you will see the name of the user that is linked to that email.

Now, head to: `https://github.com/orgs/<your_org>/people/<user_from_json_above>/sso` and you will now be able to unlink this account from the given email you want to use SAML.

**Reply by TechCharmi** [reply]
what i add in this place (<your_org>)?

**Reply by ruchi1803** [reply]
@brunojppb your answer really helped me.
